.gitignore

```diff
@@ -55,3 +55,8 @@ codecov
 # VSCode
 .vscode
 
+# ignore icon index files as they're genreated when executing
+src/old_ui/Icon/svg/index.jsx
```

Comment:
Does this have any effect on adding new icons?

Reply:
This didn't actually work, i should take this out